libraries/Update/src/HttpsOTAUpdate.h

```diff
@@ -0,0 +1,14 @@
+#include "esp_http_client.h"
+#define HttpEvent_t esp_http_client_event_t
+#define HTTPS_OTA_SUCCESS 1
+#define HTTPS_OTA_FAIL 0
+
+class HttpsOTAUpdateClass {
+
+    public:
+    void startFirmwareUpgrade(const char * cert_pem, const char * url, bool skip_cert_common_name_check);
```

Comment:
change the method name to `begin`

Reply:
does it not make more sense url to be first argument? maybe add default value to `skip_cert_common_name_check `? Again, what if I do not have the certificate? I am lame and have web server on my RaspberryPi that I want to update from. Have Letsencrypt for SSL and that is all I know. Can I update without the certificate? Like I can connect to an SSL server without certificate otherwise. Sure it's not the most secure thing, but I am just testing and not deploying commercially.

Reply:
it depends upon config option CONFIG_OTA_ALLOW_HTTP
RIght now this option is not set , Hence certificates are mandatory.

Reply:
I do not want HTTP, it's still HTTPS, but just like you browse the web and do not provide yourself a certificate for every SSL site you visit. Get it? Skip certificate verification. Is it possible?